src/base/registrations/New.svelte

```diff
@@ -69,17 +69,17 @@
   </span>
 
   <span slot="subtitle">
-    {subdomain}.{config.registrar.domain}
+    {name}.{config.registrar.domain}
   </span>
 
   <span slot="body">
     {#if state === State.NameAvailable}
       {#if registrationOwner}
-        The name <strong>{subdomain}</strong> is available for registration
+        The name <strong>{name}</strong> is available for registration
         under account <strong>{formatAddress(registrationOwner)}</strong>
         for <strong>{fee} RAD</strong>.
       {:else}
-        The name <strong>{subdomain}</strong> is available
+        The name <strong>{name}</strong> is available
         for <strong>{fee} RAD</strong>.
       {/if}
     {:else if state === State.NameUnavailable}
```

Comment:
Not sure why this file and the other was changed? `subdomain` seems more correct than `name`, since it's only part of an ENS name.

Reply:
I changed it to name since for me the subdomain actually is `.radicle` in this case, and i took the name as eg `cloudhead` or `sebastinez`.
`<name>.<subdomain>.<TLD>`

Reply:
Ok, I can live with that.